docs/changelog/102379.yaml

```
@@ -0,0 +1,6 @@
+pr: 102379
+summary: Pass source query to `FieldCapabilitiesAction` (as index_filter) for better
```

I think it's better to refer to the REST API name rather than the internal action name.

```suggestion
summary: Pass source query to `_field_caps` (as `index_filter`) when deducing destination index mappings for better
```